src/restaurant_inspections_api/constants.clj

```diff
@@ -6,3 +6,4 @@
 (def production? (env/production?))
 (def db-url (env/get-env-db-url))
 (def csv-files (env/get-csv-files))
+(def *earliest-date* "2013-01-01")
```

Comment:
@marcoslhc to actually make it a constant, use `^:const` ; example: `(def ^:const pi 3.14)` 
*var* notation is for dynamic variables (dynamic binding). Assume everything that is defined with `def` is a constant, due to clojure's approach on immutability and vars.